service/src/integrationTest/kotlin/fi/espoo/evaka/attendance/MobileUnitControllerIntegrationTest.kt

```diff
@@ -0,0 +1,136 @@
+            tx.insertGeneralTestFixtures()
+            tx.insertTestDaycareGroup(DevDaycareGroup(id = groupId, daycareId = testDaycare.id, name = groupName))
+            listOf(testChild_1, testChild_2, testChild_3, testChild_4, testChild_5).forEach { child ->
+                val daycarePlacementId = PlacementId(UUID.randomUUID())
+                tx.insertTestPlacement(
+                    id = daycarePlacementId,
+                    childId = child.id,
+                    unitId = testDaycare.id,
+                    startDate = placementStart,
+                    endDate = placementEnd,
+                    type = PlacementType.PRESCHOOL_DAYCARE
+                )
+                tx.insertTestDaycareGroupPlacement(
+                    daycarePlacementId = daycarePlacementId,
+                    groupId = groupId,
+                    startDate = placementStart,
+                    endDate = placementEnd
+                )
+            }
+            tx.createMobileDeviceToUnit(mobileUser.id, testDaycare.id)
+
+            tx.insertAttendance(testChild_1.id, testDaycare.id, today, LocalTime.of(8, 30, 0))
+            tx.insertAttendance(testChild_2.id, testDaycare.id, today, LocalTime.of(9, 0, 0))
+            tx.insertAttendance(testChild_3.id, testDaycare.id, today, LocalTime.of(9, 30, 0))
+            tx.insertAttendance(testChild_4.id, testDaycare.id, today, LocalTime.of(10, 0, 0))
+            tx.insertAttendance(testChild_5.id, testDaycare.id, today, LocalTime.of(10, 30, 0))
+
+            FixtureBuilder(tx)
+                .addEmployee()
+                .withName("One", "in group 1")
+                .withGroupAccess(testDaycare.id, groupId)
+                .withScopedRole(UserRole.STAFF, testDaycare.id)
+                .saveAnd {
+                    testEmployeeId_1 = employeeId
+                }
+                .addEmployee()
+                .withName("Two", "in group 1")
+                .withGroupAccess(testDaycare.id, groupId)
+                .withScopedRole(UserRole.STAFF, testDaycare.id)
+                .saveAnd {
+                    testEmployeeId_2 = employeeId
+                }
+
+            tx.markStaffArrival(testEmployeeId_1, groupId, now.minusDays(1), BigDecimal(7.0))
+            tx.markStaffArrival(testEmployeeId_2, groupId, now.minusDays(1), BigDecimal(7.0))
+        }
+    }
+
+    @Test
+    fun `unit utilization is same in both unit info and stats`() {
+        val unitInfo = fetchUnitInfo()
+        val allUnitStats = fetchUnitStats(listOf(testDaycare.id))
+
+        assertEquals(unitInfo.utilization, allUnitStats.first().utilization)
+    }
+
+    private fun fetchUnitInfo(): UnitInfo {
+        val (_, res, result) = http.get(
+            "/mobile/units/${testDaycare.id}",
+            listOf(Pair("useRealtimeStaffAttendance", true))
+        )
+            .asUser(mobileUser)
+            .withMockedTime(now)
+            .responseObject<UnitInfo>(jsonMapper)
```

Comment:
I think it's inconsistent that this function doesn't get unit id as parameter, but `fetchUnitStats` does.